chameleon-core/src/test/java/org/hibernate/omm/query/NativeQueryTests.java

```diff
@@ -33,12 +32,11 @@ void testNativeQueryWithoutParameter() {
         });
 
         var nativeQuery = "{ aggregate: \"books\", pipeline: [" +
-                "{ $match :  { _id: { $eq: ? } } }, " +
+                "{ $match :  { _id: { $eq: 1234 } } }, " +
```

Comment:
I tried a bunch of alternative here but none worked except to remove the parameter entirely.  There must be a way...